tests/integration_tests/test_sim.py

```diff
@@ -46,17 +47,17 @@ def set_dummy_sensor_values(sim_app, device_type: DeviceType, sensor_type: Senso
 class TestFlare:
     def test_arming(self, qtbot, sim_app, device_type):
         flush_packets(sim_app, device_type)
-        assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.STATE) == 0
+        assert sim_app.rocket_data.last_value_by_device(device_type, DataEntryIds.STATE) == STATE_IDS[0x00]
```

Comment:
magic number?

Reply:
Similar situation to the first comment, also being a testing mechanism too.
OR
For this one I could make it `local variable = 0x00`?